api/src/main/java/org/pmiops/workbench/db/dao/UserDao.java

```diff
@@ -48,7 +46,21 @@
 
   Set<DbUser> findByFirstRegistrationCompletionTimeNotNull();
 
-  default Map<Boolean, Long> getDisabledToCountMap() {
-    return DaoUtils.getAttributeToCountMap(findAll(), DbUser::getDisabled);
+  @Query(
+      "SELECT dataAccessLevel, disabled, CASE WHEN betaAccessBypassTime IS NOT NULL THEN TRUE ELSE FALSE END AS betaIsBypassed, COUNT(userId) AS userCount "
```

Comment:
One way around this ugliness is a persisted, computed column. Not worth it here, as much as I hate seeing this comparison to null everywhere to essentially build a boolean.